Doc/library/os.path.rst

```diff
@@ -266,6 +266,14 @@ the :mod:`glob` module.)
       Accepts a :term:`path-like object`.
 
 
+.. function:: isjunction(path)
+
+   Return ``True`` if *path* is a junction. This function exists for both POSIX
+   and Windows, though can only return True on Windows.
```

Comment:
```suggestion
   and Windows, though only Windows currently supports junctions.
```

This allows us to support junctions on other platforms if it ever comes up. Making the details too concrete would mean we'd have to deprecate the function before allowing another platform to return True for their junctions.